packages/react-app/src/components/history/BridgeHistory.jsx

```diff
@@ -45,8 +58,13 @@ export const BridgeHistory = ({ page }) => {
       px={{ base: 4, sm: 8 }}
       w="100%"
     >
+      <ClaimErrorModal
```

Comment:
You will have to show this error even in `components/modals/ClaimTransferModal.jsx`

Reply:
But for this case we may not need to show the modal as such but just show a toast